config/locales/vi.yml

```
@@ -34,15 +34,15 @@ vi:
                     <a href="https://www.ra Whileutorial.org/contact"> trang liên hệ </a>.
                   </ p>
   base_title: Hướng dẫn ruby on rails ứng dụng đơn giản
-  footer_html: <small>
-                  <a href="https://www.railstutorial.org/">Hướng Dẫn Ruby On Rails</a>
-                  bởi <a href="http://www.michaelhartl.com/">Michael Hartl</a>
-                </small>
-  footer1_html: <li><a href="http://news.railstutorial.org/">Tin tức</a></li>
-  header_html:  <li><%= link_to "Trang chủ",   "#" %></li>
-                <li><%= link_to "Trợ giúp",   "#" %></li>
-                <li><%= link_to "Đăng nhập", "#" %></li>
-  footer2_html: <li><%= link_to "Hướng dẫn",   "# %></li>
-                <li><%= link_to "Liên hệ", "#" %></li>
+  footer: Hướng dẫn Ruby on Rails bởi Michael Hartl
+  News: Tin tức
```

fix chưa hết nè em, comment 1 chỗ thì phải fix những chỗ khác bị, không chờ bị comment dòng nào rồi mới fix dòng đó nha em
```suggestion
  news: Tin tức
```